packages/expo-notifications/android/src/androidTest/kotlin/expo/modules/notifications/NotificationTriggerTest.kt

```diff
@@ -1,4 +1,3 @@
-@file:Suppress("DEPRECATION")
```

Comment:
I didn't see any deprecation warnings when I remove this but I may have missed something